### PR TITLE
Include spock in the server programming section.

### PR DESCRIPTION
**Pull request by cgag**

### CODE REVIEW

File: sotu.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ of the Haskell runtime.
 
 * [`warp`](https://hackage.haskell.org/package/warp) / [`wai`](https://hackage.haskell.org/package/wai) - the low-level server and API that all server libraries share, with the exception of `snap`
 * [`scotty`](https://hackage.haskell.org/package/scotty) - A beginner-friendly server framework analogous to Ruby's Sinatra
+* [`spock`](https://www.spock.li/) - Lighter than the "enterprise" frameworks, but more featureful than spock (type-safe routing, sessions, conn pooling, csrf protection, authentication, etc)
```
Comment (GetContented) [inline sotu.md]: This doesn't read correctly. Is it supposed to say "more featureful than scotty"?

Comment (Gabriella439) [inline sotu.md]: Yeah, I meant "more featureful than scotty"

Comment (Gabriella439) [inline sotu.md]: Oops, I didn't realize that I didn't write that :blush:

Comment (cgag) [inline sotu.md]: haha, yeah, i meant to write "more featureful than scotty", woops

**Comment by Gabriella439** [comment]
Thank you for adding this!

**Pull request merged**